doc/api/corepack.md

```diff
@@ -12,10 +12,8 @@ added:
 
 > Stability: 1 - Experimental
 
-Documentation for this tool can be found on the [Corepack repository][].
+**Corepack will no longer be distributed with Node.js 25+.**
+Users currently depending on the bundled `corepack` module from Node.js
```

Comment:
nit: introduce a newline so that there's a separation between the decision in bold, and call to action for existing users.
```suggestion
**Corepack will no longer be distributed with Node.js 25+.**

Users currently depending on the bundled `corepack` module from Node.js
```